conan/tools/cmake/cmake.py

```diff
@@ -32,13 +32,12 @@ def _cmake_cmd_line_args(conanfile, generator):
 
 
 class CMake(object):
-    """ CMake helper to use together with the toolchain feature. It implements a very simple
-    wrapper to call the cmake executable, but without passing compile flags, preprocessor
-    definitions... all that is set by the toolchain. Only the generator and the CMAKE_TOOLCHAIN_FILE
-    are passed to the command line, plus the ``--config Release`` for builds in multi-config
-    """
+    """ CMake helper to use together with the CMakeToolchain feature """
```

Comment:
Because the basic description of this class goes to the ``.rst`` file? Or who will explain that this class implements the command line wrapper over the cmake executable?

Reply:
It is explained on the introduction page, before the reference. This docstring indeed goes to the rst.